server/JsDbg.Gdb/JsDbgBase.py

```diff
@@ -0,0 +1,138 @@
+# This is the non-debugger specific base functionality of JsDbg.
```

Comment:
For this to be used by both GDB and LLDB, I think we have to move this out of JsDbg.GDB and into a location that both the GDB and LLDB servers can refer to. We do something similar to share core functionality between JsDbg.WinDbg and JsDbg.VisualStudio.

Reply:
Do you mind if I do that in a separate patch? Because I really should move the webserver code as well, and that would make this a fairly big patch.

For the other question -- this is a future-looking change

Reply:
Actually I'll just do a second commit on this PR. Will let you know when I'm done.

Reply:
OK, please take another look